packages/manager-components/src/components/templates/update-name-modal/update-name-modal.stories.tsx

```diff
@@ -31,6 +33,9 @@ const meta: Meta = {
     cancelButtonLabel: 'Cancel',
     confirmButtonLabel: 'Confirm',
     error: '',
+    pattern: '^[a-zA-Z0-9-_s]*$',
+    patternMessage:
+      "Le nom du service doit comporter entre 3 et 63 caractères , doit commencer et se terminer par des caractères alphanumériques minuscules (de a à z et de 0 à 9) et peut comporter les signes de ponctuation suivants : '_', '.' et '-'.",
```

Comment:
**suggestion:** Since mostly the messages are translated, maybe we can add the example like below for better understanding,
```suggestion
    patternMessage: t('pattern_error_message'),
```

Reply:
the message error is from PCI rancher project, and its very specific to this project.
IMO, We can not maintain all translations in this component

Reply:
But I can put a translations mocked